src/vue-components/lookup.vue

```diff
@@ -113,7 +126,13 @@
         if (this.lookuptext.length === 0) {
           return null
         }
-        let languageID = LanguageModelFactory.getLanguageIdFromCode(this.options.items.lookupLanguage.currentValue)
+        let languageID
```

Comment:
Maybe to replace this with ternary operator?

Reply:
I thought about it - bit it will be really long line with both variants - and it won't be easily to read.
That's wjy I chose this variant as it more readable.

Reply:
We could record it as multiline according to Standard rules (https://standardjs.com/rules.html)
```
    const languageID = this.overrideLanguage
      ? languageID = LanguageModelFactory.getLanguageIdFromCode(this.lookupLanguage.currentValue)
      : languageID = LanguageModelFactory.getLanguageIdFromCode(this.options.items.lookupLanguage.currentValue)
```
But it's probably a matter of personal preference so I do not insist :)